docs/source/contents/4_draft_mapping.rst

```diff
@@ -372,15 +381,15 @@ Samvera uses brackets to wrap title strings in direct mapping examples. Accordin
 provided by DPLA, they recommend we "not have brackets or ending periods."
 
 
-Xpath
+XPath
 ^^^^^
 
-`mods:titleInfo[@supplied="yes"]/mods:title`
+:code:`titleInfo[@supplied="yes"]/title`
 
 Decision
 ^^^^^^^^
 
-In these cases a supplied="yes" may also be present for one titleInfo element. Supplied titles would be used as dcterms:title. Triples will not indicate supplied titles using brackets.
+In these cases a :code:`supplied="yes"` may also be present for one :code:`titleInfo` element. Supplied titles would be used as `dcterms:title`. Triples will not indicate supplied titles using brackets.
```

Comment:
```suggestion
In these cases a :code:`supplied="yes"` may also be present for one :code:`titleInfo` element. Supplied titles would be used as  :code:`dcterms:title`. Triples will not indicate supplied titles using brackets.
```

Reply:
I made a-possibly-ill-considered stylistic decision: properties will be only be backticked and XPaths will be inline-code-prefixed; i.e. `` `dbo:isPartOf` `` and `` :code:`titleInfo[\@supplied='yes']` ``.

Does that make sense? Happy to discuss alternates if you'd like.

Reply:
Yeah, that makes sense. Thanks.